pkg/nats/proxy/compute_proxy.go

```diff
@@ -50,10 +52,10 @@ func (p *ComputeProxy) CancelExecution(
 		ctx, p.conn, request.TargetPeerID, CancelExecution, request)
 }
 
-func (p *ComputeProxy) ExecutionLogs(
-	ctx context.Context, request compute.ExecutionLogsRequest) (compute.ExecutionLogsResponse, error) {
-	return proxyRequest[compute.ExecutionLogsRequest, compute.ExecutionLogsResponse](
-		ctx, p.conn, request.TargetPeerID, ExecutionLogs, request)
+func (p *ComputeProxy) ExecutionLogs(ctx context.Context, request compute.ExecutionLogsRequest) (
+	<-chan *concurrency.AsyncResult[models.ExecutionLog], error) {
+	//TODO implement me
```

Comment:
Is this coming in a follow up PR or a later commit to this one?

Reply:
yeah in a follow up PR. I have an open issue for that https://github.com/bacalhau-project/expanso-planning/issues/413